### PR TITLE
Feat/signin

### PR DESCRIPTION
**Pull request by hcnimi**
includes webpack and updated package.json.

### CODE REVIEW

File: client/src/components/SignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+class SignIn extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange = event => {
```
Comment (skoshkarli) [inline client/src/components/SignIn.jsx]: should event have parenthesis around it?

File: client/src/components/SignIn.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    });
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+    // TODO: route
+  }
+
+  render() {
+    render() {
```
Comment (skoshkarli) [inline client/src/components/SignIn.jsx]: render inside render

File: client/src/components/SignIn.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    return (
+      <div className='login'>
+        <form onSubmit={this.handleSubmit}>
+          <FormGroup controlId='email' bsSize='large'>
+            <ControlLabel>Email</ControlLabel>
+            <FormControl
+              autoFocus
+              type='email'
+              value={this.state.email}
+              onChange={this.handleChange}
```
Comment (skoshkarli) [inline client/src/components/SignIn.jsx]: do we need to bind handleChange to this in state?

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  next();
+});
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+// app.post('', (req, res) => {
+
+// });
+
+app.get('/signin', (req, res) => {
```
Comment (skoshkarli) [inline server/index.js]: should signin be a post request?

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+// app.post('', (req, res) => {
```
Comment (skoshkarli) [inline server/index.js]: we should remove any unused code without TODO's

File: client/src/components/SignIn.jsx (hunk 1 above)
Comment (hcnimi) [reply client/src/components/SignIn.jsx]: if only 1 argument, parens not needed

File: client/src/components/SignIn.jsx (hunk 2 above)
Comment (hcnimi) [reply client/src/components/SignIn.jsx]: good catch.

**Comment by skoshkarli** [comment]
🚢 it!

**Pull request merged**